Remove QuantumScript.is_sampled and QuantumScript.all_sampled (#5072)
PennyLaneAI/pennylane: doc/development/deprecations.rst

  - Deprecated in v0.34
  - Removed in v0.35

> * ``QuantumScript.is_sampled`` and ``QuantumScript.all_sampled`` are deprecated. Users should now validate

Reviewer: ```suggestion
* ``QuantumScript.is_sampled`` and ``QuantumScript.all_sampled`` have been removed. Users should now validate
```
